sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -1,37 +1,153 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
 
 package com.azure.security.attestation;
 
 import com.azure.core.annotation.ReturnType;
 import com.azure.core.annotation.ServiceClient;
 import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.exception.HttpResponseException;
 import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
 import com.azure.core.util.Context;
 import com.azure.core.util.FluxUtil;
+import com.azure.core.util.logging.ClientLogger;
 import com.azure.security.attestation.implementation.AttestationClientImpl;
 import com.azure.security.attestation.implementation.AttestationsImpl;
 import com.azure.security.attestation.implementation.MetadataConfigurationsImpl;
 import com.azure.security.attestation.implementation.SigningCertificatesImpl;
-import com.azure.security.attestation.models.AttestOpenEnclaveRequest;
-import com.azure.security.attestation.models.AttestSgxEnclaveRequest;
-import com.azure.security.attestation.models.AttestationResponse;
+import com.azure.security.attestation.implementation.models.AttestationOpenIdMetadataImpl;
+import com.azure.security.attestation.implementation.models.AttestationOptionsImpl;
+import com.azure.security.attestation.implementation.models.AttestationResultImpl;
+import com.azure.security.attestation.implementation.models.AttestationSignerImpl;
+import com.azure.security.attestation.implementation.models.AttestationTokenImpl;
+import com.azure.security.attestation.models.AttestationOpenIdMetadata;
+import com.azure.security.attestation.models.AttestationOptions;
+import com.azure.security.attestation.models.AttestationResult;
 import com.azure.security.attestation.models.AttestationSigner;
-import com.azure.security.attestation.models.CloudErrorException;
+import com.azure.security.attestation.models.AttestationToken;
 import reactor.core.publisher.Mono;
 
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
 
 import static com.azure.core.util.FluxUtil.withContext;
 
-/** Initializes a new instance of the asynchronous AzureAttestationRestClient type. */
+/**
+ * The AttestationAsyncClient implements the functionality required by the "Attest" family of APIs.
```

Comment:
ApiView updated. No idea why it wasn't there, but it is there now.

Reply:
There's nothing in target\generated-sources. Where should I look for it? It's in the target\apidocs directory.

Reply:
Should I be uploading the javadoc.jar or the sources.jar file to ApiView? I've been uploading the sources.jar because that's what APiView said I should upload.

Reply:
Answer: Nope, that breaks ApiView...

Reply:
Apparently ApiView doesn't like it if there is a line between the trailing comment and the definition of the class.

Removed the blank line and it's fine.